docs/01_nodeos/03_plugins/net_plugin/index.md

```diff
@@ -35,16 +35,16 @@ Config Options for eosio::net_plugin:
                                         indicates to node that only
                                         transactions 'trx' or blocks 'blk'
                                         should be sent.  Examples:
-                                            p2p.example.org:9876
-                                            p2p.trx.example.org:9876:trx
-                                            p2p.blk.example.org:9876:blk
+                                            p2p.eos.io:9876
+                                            p2p.trx.eos.io:9876:trx
+                                            p2p.blk.eos.io:9876:blk
```

Comment:
I don't have strong feelings here. [Previously this was changed ](https://github.com/eosnetworkfoundation/mandel/pull/748/files#diff-5cd6d2d6d53259f0b62b7e22b7baf9951b24232907c8d4bb5ea17929199206f5) to example.org to comply with [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.txt) see 2.

Reply:
If the *.eos.io is not a real endpoing, then better to use example.org.

Reply:
changed to example.org and created alternate PR #583